### ISSUE TITLE
Start getting error "Logseq is syncing internal status, please wait for several seconds" and Can not close/exite Logseq,..

### ISSUE THREAD

**Issue by myownstandard** [haydenull/logseq-plugin-excalidraw#11]
### Search first

- [X] I searched and no similar issues were found

### What Happened?

Right after I start using Excalidraw plugin regulary this issue began to happen if i create Excalidraw file during my Logseq session.
When i click a close/exit Logseq i am getting this message/error.
No matter what page is open simple Page, Journal page or Whiteboard. 
Untill I manually click Re-Index graph i am getting this error and can not exit Logseq.
I am not quite sure 100% if it is Logseqs error or Plugins issue, but i am getting it from time to time after installing Excalidraw Plugin...
Thank you!

### Reproduce the Bug

1. Open Logseq
2. Create /Excalidraw: Create New Drawing -> draw some lines/text/rectangles
3. Save and Exit Drawing
4. Then try to exit Logseq

### Screenshots

![Logseq_NALI56wLnn](https://github.com/haydenull/logseq-plugin-excalidraw/assets/78643199/45cc15c5-79b7-426f-a3a3-f86051c43e88)

### Environment Information

Windows 10, Logseq app 0.9.8 , plugin Excalidraw 1.3.0

### Are you willing to submit a PR? If you know how to fix the bug.

- [ ] I'm willing to submit a PR (Thank you!)

**Comment by haydenull** [comment]
Than you for your feedback. I have already reported this issue to the logseq team. https://github.com/logseq/logseq/issues/9612

**Comment by morrolinux** [comment]
Still happening here. If I force-close the drawing is lost.
I don't know if it's possible to implement in a similar way, but I think the drawio plugin did this right.

**Comment by ElectricBoot** [comment]
Yes, I want to add that I am also facing this issue (exactly same how OP has described it). OS: windows 11 23H2, Logseq: 0.10.9, excalidraw plugin version: v1.5.1

But after force closing (end task via task manager), I don't lose any drawing/work I did in that logseq session.

Till this issue is resolved, what should I do in such situations? End task in task manager or reindex the graph? 
The former is a fast and convenient option whereas the latter is time taking & resource intensive option. (I have a big graph). My concern with the first option is: How probable data loss is if I close logseq by ending task?
